DatabaseHandler/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+namespace DatabaseHandler
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using(Maplestory2DBEntities context = new Maplestory2DBEntities()) 
```

Comment:
The Main method for this solution should probably create the DB and then add some test data. It's adding the test data but does running this method also set up the DB?

Reply:
This is for manually adding test data. Its a console .cs file.

Reply:
Sure, but then what does somebody need to run to get the DB built themselves?

Reply:
This comes with the solution when databasehandler solution was made. They can still generate the DB via the script file i included and generate their own .edmx file or create their own ORM link.